modules/vulnerabilities/unix/local/gnuscreen/secgen_metadata.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+
+<vulnerability xmlns="http://www.github/cliffe/SecGen/vulnerability"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="http://www.github/cliffe/SecGen/vulnerability">
+  <name>Gnu Screen 4.5.0 Priv esc</name>
+  <author>James Davis</author>
+  <module_license>MIT</module_license>
+  <description>
+    GNU screen before 4.5.1 allows local users to modify arbitrary files and consequently gain root privileges by leveraging improper checking of logfile permissions.
+  </description>
+
+  <type>local_priv_escal</type>
+  <privilege>root_rwx</privilege>
+  <access>local</access>
+  <platform>linux</platform>
+  <difficulty>medium</difficulty>
+
+  <read_fact>strings_to_leak</read_fact>
+  <read_fact>leaked_filenames</read_fact>
+
+  <default_input into="strings_to_leak">
+    <generator type="flag_generator"/>
+  </default_input>
+
+  <default_input into="leaked_filenames">
+    <generator type="filename_generator"/>
+  </default_input>
+
+  <cve>CVE-2017-5618</cve>
+  <cvss_base_score>7</cvss_base_score>
+  <cvss_vector>AV:N/AC:L/Au:N/C:C/I:C/A:C</cvss_vector>
+  <reference>https://nvd.nist.gov/vuln/detail/CVE-2017-5618</reference>
+  <reference>https://www.exploit-db.com/exploits/41154</reference>
+  <software_name>gnuscreen</software_name>
+  <software_license>GPLv2</software_license>
+
+  <conflict>
+    <name>.*</name>
```

Comment:
This would conflict with everything (disabling this new module). Could you please update and test the metadata is correct.

Reply:
You are right. Not sure how that managed to creep in there. I have updated this now to correctly reflect the base conflicts.